nft/nft.go

```diff
@@ -98,13 +106,7 @@ func (hs NFTHash) IsValid([]byte) error {
 }
 
 var (
-	CopyrighterType   = hint.Type("mitum-nft-copyrighter")
-	CopyrighterHint   = hint.NewHint(CopyrighterType, "v0.0.1")
-	CopyrighterHinter = NFT{BaseHinter: hint.NewBaseHinter(CopyrighterHint)}
-)
-
-var (
-	NFTType   = hint.Type("mitum-nft-post-info")
+	NFTType   = hint.Type("mitum-nft-nft")
```

Comment:
@wyuinche 
It seems more accurate to name the NFT type as Token type. Since the NFT model deals with non-fungible tokens, it seems to be a token type that is handled in the NFT model.
Then it would be better to name the hint type as mitum-nft-token.

Reply:
I think it would be more accurate to rename the NFT type to Token type.

Reply:
@wyuinche
There seems to be a difference of opinion about the name of hint and the name of NFT type.
For faster progress, we will talk about the hint name and the NFT type name later, and I only request the agentBox part.
And you can add @yoonilc as a reviewer too.